yoti-sdk-impl/src/test/java/com/yoti/api/client/shareurl/extension/SimpleThirdPartyAttributeExtensionBuilderTest.java

```diff
@@ -0,0 +1,138 @@
+package com.yoti.api.client.shareurl.extension;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.yoti.api.client.AttributeDefinition;
+import com.yoti.api.client.spi.remote.call.YotiConstants;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+public class SimpleThirdPartyAttributeExtensionBuilderTest {
+
+    public static final Date SOME_DATE = new Date();
+    public static final String SOME_DEFINITION = "com.thirdparty.id";
```

Comment:
These don't need to be public do they?

Reply:
Nope, good spot.  Will change these 👍

Reply:
Fixed in https://github.com/getyoti/yoti-java-sdk/pull/142/commits/04fe75becb7960203ed6e759fb81251c010bf64d